src/LottoCalculator.js

```diff
@@ -0,0 +1,74 @@
+import { Console } from '@woowacourse/mission-utils';
+import { OUTPUT_MESSAGE } from './lib/constant.js';
+
+class LottoCalculator {
+  #result = {
+    three: 0,
+    four: 0,
+    five: 0,
+    fiveBonus: 0,
+    six: 0,
+  };
+
+  #ROI = 0;
+
+  constructor(buyPrice, lottos, winningNumbers, bonusNumber) {
+    this.buyPrice = buyPrice;
+    this.lottos = lottos;
+    this.winningNumbers = winningNumbers;
+    this.bonusNumber = bonusNumber;
+    this.calculateResult(lottos, winningNumbers, bonusNumber);
+    this.calculateROI(buyPrice);
```

Comment:
여기서 `buyPrice`, `lottos`, `winningNumber`, `bonusNumber`는 해당 클래스에서 쓰이는 상태가 아니니 `const`로 변수 선언을 해도 괜찮지 않을까요?


특히 `buyPrice`와 `lottos`는 다른 메서드에서 쓰이지 않고, `calculateResult`와 `calculateROI`의 파라미터로만 전달되는 것 같습니다.

```suggestion
    const winningNumbers = winningNumbers;
    const bonusNumber = bonusNumber;
    this.calculateResult(lottos, winningNumbers, bonusNumber);
    this.calculateROI(buyPrice);
```

Reply:
https://github.com/woowacourse-precourse/javascript-lotto-7/pull/55#discussion_r1830404877
요기와 같은 문제점이네요! 다음엔 신경써보겠습니다! 😎